packages/common/src/utils/index.ts

```diff
@@ -40,6 +40,26 @@ export const generateRandomNumber = (): number => {
   return Math.floor(Math.random() * 1e6);
 };
 
+/**
+ * Pseudo generate random numbers giving a maximum
+ * @disclaimer Not suitable for crypto.
+ * @param maximum maximum number that the random number can be
+ * @returns a number from 0 to maximum number
+ */
+export const generatePseudoRandomId = (maximum: number) => {
+  return Math.floor(Math.random() * maximum);
+};
+
+/**
+ * Choose pseudo random element from an array
+ * @disclaimer Not suitable for crypto.
+ * @param arr array to choose random element
+ * @returns element from array
+ */
+export const randomlySelectFromArray = <T>(arr: T[]): T => {
+  return arr?.[Math.floor(Math.random() * arr.length)];
```

Comment:
should error if array is empty, this function should be called when there are contents in the array

Reply:
in the places where this function is used, it should check if the array is empty and handle it